src/ASM/GlbL2projector.C

```diff
@@ -33,6 +34,103 @@ LinAlg::MatrixType GlbL2::MatrixType   = LinAlg::SPARSE;
 LinSolParams*      GlbL2::SolverParams = nullptr;
 
 
+namespace {
+
+static void expandTensorGrid2 (const RealArray* in, RealArray* out)
```

Comment:
Is it necessary with the anonymous `namespace` scoping here? If declaring the functions static, they will be "visible" only for the code in this file in any case (at least, that's what I thought).

Reply:
static functions is the 'C' way to hide functions. anonymous namespace is the C++ way to do it. just gotten used to doing the latter as of late, but i can of course do it static functions instead if you prefer.

Reply:
OK, but when you do both, like here, it looks like "smør på flesk"/confusing. I have always used the "static" approach elsewhere in the code, so the only reason I can see is consistency, if the result is equivalent. But you can use namespace if you prefer that.

Reply:
i prefer consistency, so made them static. the reason for the combination is that these are copy-pasted from elsewhere (but i did not want to expose them to global namespace..). updated one, forgot the other.